example/particle_tracers/particle_tracers.cpp

```diff
@@ -146,10 +148,10 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
 
   // Add swarm of tracer particles
   std::string swarm_name = "tracers";
-  Metadata swarm_metadata({Metadata::Provides, Metadata::None});
+  Metadata swarm_metadata(
+      {Metadata::Provides, Metadata::None, Metadata::NoPersistentParticleIds});
```

Comment:
Why is the `id` field being removed here when previously the example explicitly created an `id` field?